app/scripts/templates/permissions.mustache

```diff
@@ -1,20 +1,22 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="fxa-permissions-header">{{#t}}%(serviceName)s would like to know…{{/t}}</h1>
+    <h1 id="fxa-permissions-header">{{#t}}Request for permission{{/t}}</h1>
   </header>
 
   <section>
     <div class="error"></div>
     <div class="success"></div>
 
-    <p id="permissions">
-      <span class="permission">{{#t}}Email{{/t}}</span>
-      <span class="value">{{ email }}</span>
-    </p>
+    <p>{{#t}}%(serviceName)s wants to use the following info:{{/t}}</p>
+
+    <form id="permissions" novalidate>
+      <div class="input-row">
+        <label class="label-helper visible">{{#t}}Email{{/t}}</label>
```

Comment:
`label-helper` is meant to enable placeholders to [slide up when an input element is focused](https://github.com/mozilla/fxa-content-server/blob/9016164b7742c04ddf15169c1ccceac2a2634941/app/scripts/views/mixins/floating-placeholder-mixin.js#L31). Since the input element is disabled, that doesn't really make sense.

Instead of overloading `label-helper`'s usage for styling, what about just removing the classes and [updating the label's style directly](https://github.com/mozilla/fxa-content-server/pull/2442/files#diff-79e8639722018aa387b343afe8e4b56fR135).